hiive/mdptoolbox/example.py

```diff
@@ -402,3 +404,62 @@ def small():
     P = _np.array([[[0.5, 0.5], [0.8, 0.2]], [[0, 1], [0.1, 0.9]]])
     R = _np.array([[5, 10], [-1, 2]])
     return P, R
+
+def openai(env_name:str, render:bool=False, **kwargs):
+    """
+    Generate a MDPToolbox-formatted version of a *discrete* OpenAI Gym environment. 
+
+    You can find the list of available gym environments here: https://gym.openai.com/envs/#classic_control
+
+    You'll have to look at the source code of the environments for available kwargs; as it is not well documented.  
+
+    This function is used to generate a transition probability
+    (``A`` × ``S`` × ``S``) array ``P`` and a reward (``S`` × ``A``) matrix
+    ``R``.
+
+    Parameters
+    ---------
+    env_name : str
+        The name of the Open AI gy environment to model. 
```

Comment:
small typo - gy --> gym

Reply:
Fixed in 952d47ab5aebeef39051df22c677d494c990c031